docs/user/integration/sample-app/README.md

```diff
@@ -0,0 +1,122 @@
+
+### Traces
+
+- The `main` method initializes a tracer and a propagator
+- In the handler routines the tracer is used to create new spans
+- The propagation in the `forward` method is done automatically by passing the request context to the upstream call
+- In the `main` method, the handler functions are getting auto-instrumented using the [`otelhttp`](https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp) library, having request spans auto-instrumented having span names prefixed with `auto-`
+
+### Logs
+
+The `main.go` initializes the Golang `slog` logger, which is consistently used for all application logs. It is configured to print in JSON and is available the logs will add the traceId attribute.
+
+### Running local
+
+By default the exporter are configured to print to stdout, so that you can run the app from local.
+
+1. Checkout the `telemetry-manager` repo and browse into the folder `docs/user/integration/sample-app`
+1. Build and start the application:
+    ```sh
+    make run
+    ```
+
+## Deploying the Sample App
+
+### Activate Kyma Telemetry with a backend
+1. Provide a tracing backend and activate it.
+   Install [Jaeger in-cluster](../jaeger/README.md) or provide a custom backend supporting the OTLP protocol (like [SAP Cloud Logging](./../sap-cloud-logging/)).
+1. Provide a metric backend and activate it.
+   <!-- markdown-link-check-disable-next-line -->
+   Install [Prometheus in-cluster](../prometheus/README.md) or provide a custom backend supporting the OTLP protocol (like [SAP Cloud Logging](./../sap-cloud-logging/)).
+1. Provide a log backend and activate it.
+   Install [Loki in-cluster](../loki/README.md) or provide a custom backend supporting the OTLP protocol (like [SAP Cloud Logging](./../sap-cloud-logging/)).
+
+### Deploy the sample application
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export K8S_SAMPLE_NAMESPACE="{namespace}"
+    ```
+
+1. Ensure that your Namespace has Istio sidecar injection enabled to have a secure communication enabled by default:
+
+   ```bash
+   kubectl label namespace ${K8S_SAMPLE_NAMESPACE} istio-injection=enabled
+   ```
+
+1. Deploy the service using the prepared Deployment manifest and image:
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/docs/user/integration/sample-app/deployment/deployment.yaml -n $K8S_SAMPLE_NAMESPACE
+    ```
+
+1. Verify the application:
+
+   Port-forward to the service:
+   ```sh
+   kubectl -n $K8S_SAMPLE_NAMESPACE port-forward svc/sample-app 8080
+   ```
+   and call the forward endpoint:
+   ```sh
+   curl http://localhost:8080/forward
+   ```
+
+### Cleanup
+
+Run the following commands to completely remove the sample app from the cluster:

```

Comment:
line 116 and 118 seem redundant. Can we drop one?

Reply:
fixed